policy/modules/system/systemd.te

```diff
@@ -178,6 +178,7 @@ type systemd_networkd_exec_t;
 init_system_domain(systemd_networkd_t, systemd_networkd_exec_t)
 
 type systemd_networkd_runtime_t alias systemd_networkd_var_run_t;
+files_mountpoint(systemd_networkd_runtime_t)
```

Comment:
This is one of those things. Not sure if we reached any consensus on this topic. but personally i would argue that these aren't your average mountpoints and as such shouldnt be treated that way

The only thing ever mounting on systemd_networkd_runtime_t type directories will ever be systemd.

This rule is too broad in my view

Reply:
An alternative would be a direct rule granting `init_t` rights to `mounton` on `systemd_networkd_runtime_t`. This would need to go via an interface and be conditional on `init_systemd`.

Reply:
Or maybe a better one: declare an attribute `systemd_nspawn_mountpoint` and allow systemd's `init_t` `mounton` rights to it. This would allow for future extension.

Reply:
Scratch that - there is an interface for that already: `init_mountpoint`.

Reply:
I guess "systemd_mountpoint_type" should be suffice. It is not only for nspawn.

Reply:
as a matter of fact, nspawn has little to do with this AFAIK

Reply:
oh ok, yes init_mountpoint sounds appropriate